src/components/Badge/Badge.stories.mdx

```diff
@@ -0,0 +1,255 @@
+import { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import Badge from './Badge';
+import Box from '../Box/Box';
+import Button from '../Button/Button';
+
+<Meta
+  title="Components/Badge"
+  component={Badge}
+/>
+
+# Badge
+
+Use badge to display feedback for users about specific actions, or states of an application. All that is required to render a basic version of an Badge is a `message` prop which will contain text
+or a ReactNode to be rendered.
+
+<Canvas isExpanded>
+  <Box direction="row">
+    <Badge
+      message="Hello world!"
+    />
+  </Box>
+</Canvas>
+
+## Props
+
+<ArgsTable of={Badge} />
+
+
+## Variants
+
+Badges come in different variants based on the kind of feedback being delivered to the user.
+
+<Canvas>
+  <Story name="Variants">
+    {() => {
+      const variants = [
+        'default',
+        'info',
+        'tertiary',
+        'success',
+        'warning',
+        'danger',
+      ];
+      return (
+        <Box direction="row">
+         {variants.map(variant => <Badge message={variant} variant={variant} />)}
+        </Box>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Sizes
+
+<Canvas>
+  <Story name="Sizes">
+    {() => {
+      return (
+        <>
+          <Box direction="row" className="m-bottom-md">
+            <Badge
+              message="Small"
+              size="sm"
+            />
+          </Box>
+          <Box direction="row" className="m-bottom-md">
+            <Badge
+              message="Base"
+              size="md"
+            />
+          </Box>
+          <Box direction="row" className="m-bottom-md">
+            <Badge
+              message="Large"
+              size="lg"
+            />
+          </Box>
+          <Box direction="row">
+            <Badge
+              message="X-Large"
+              size="xl"
+            />
+          </Box>
+        </>
+      )
+    }}
+  </Story>
+</Canvas>
+
+
+## Custom Class
+
+You can apply a custom class to your badge with the `className` prop. Here we have added a 3xl bottom margin.
+
+<Canvas>
+  <Story name="Custom Class">
+    {() => {
+      return (
+        <Box direction="row">
+          <Badge
+            message="Hello world!"
+            variant="success"
+            className="m-bottom-3xl"
+          />
+        </Box>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## With Icon
+
+You can choose to display a standard icon for each variant of notification, based on their nature.
+
+<Canvas>
+  <Story name="With Icon">
+    {() => {
+      const variants = [
+        'default',
+        'info',
+        'tertiary',
+        'success',
+        'warning',
+        'danger',
+      ];
+      const message = (variant) => `
```

Comment:
Badges shouldn't be used to display messages like this, that's what Alerts are for.